src/components/OurCourses/index.js

```diff
@@ -0,0 +1,58 @@
+import { Typography, useMediaQuery, Grid } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import OurCourseCard from "@/pesayetu/components/OurCourseCard";
+import Section from "@/pesayetu/components/Section";
+
+const OurCourses = ({ title, items, ...props }) => {
+  let itemsData = [];
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+
+  itemsData = isTablet ? items.slice(0, 2) : items;
```

Comment:
Missing carousel on mobile design.. and that should apply on tablets too.

Reply:
Done